game.py

```diff
@@ -89,7 +90,7 @@ def __init__(self, canvas, parent=None, path=None):
 
         self._width = Gdk.Screen.width()
         self._height = Gdk.Screen.height()
-        self._scale = self._width / 1200.
+        self._scale = self._width // 1200.
```

Comment:
As one of the values here is a float, use `/` and not `//` so as to use the same values that were used before the port.